src/compressed_tensors/utils/match.py

```diff
@@ -101,6 +111,29 @@ def match_named_parameters(
             )
 
 
+def match_targets(
+    name: str, module: torch.nn.Module, targets: Iterable[str]
+) -> List[str]:
+    """
+    Returns the targets that match the given name and module.
+    Outputs are ordered by type: exact name match, regex name match, class name match
+    """
+    if isinstance(module, InternalModule):
+        return []
+
+    targets = sorted(targets, key=lambda x: ("re:" in x, x))
```

Comment:
Is there a reason why this should be sorted?

Reply:
See response above.